composer.json

```diff
@@ -85,9 +85,7 @@
             "./bin/console lint:yaml ./config",
             "./bin/console lint:yaml .travis.yml",
             "./vendor/bin/phpcs -s .",
-            "./vendor/bin/minus-x check .",
-            "./node_modules/.bin/lessc --lint assets/*.less",
-            "./node_modules/.bin/eslint ."
+            "./vendor/bin/minus-x check ."
```

Comment:
I think you can remove `./vendor/bin` and just do `minus-x check .` (same with the line above)

Reply:
Oh yeah! :) Done.